native-sql-engine/core/src/main/scala/com/intel/oap/execution/ColumnarSortExec.scala

```diff
@@ -94,17 +94,37 @@ case class ColumnarSortExec(
   def buildCheck(): Unit = {
     // check types
     for (attr <- output) {
-      try {
-        ConverterUtils.checkIfTypeSupported(attr.dataType)
-      } catch {
-        case e: UnsupportedOperationException =>
+      attr.dataType match {
+        case d: BooleanType =>
+        case d: ByteType =>
+        case d: ShortType =>
+        case d: IntegerType =>
+        case d: LongType =>
+        case d: FloatType =>
+        case d: DoubleType =>
+        case d: StringType =>
+        case d: DateType =>
+        case d: DecimalType =>
+        case d: TimestampType =>
+        case d: ArrayType =>
+        case d: MapType =>
+        case d: StructType =>
+        case _ =>
```

Comment:
checkIfNestTypeSupported?

Reply:
checkIfNestTypeSupported is for non-nest.